packages/context/src/__tests__/unit/context-view.unit.ts

```diff
@@ -43,6 +46,21 @@ describe('ContextView', () => {
     expect(view.bindings).to.eql([bindings[1], bindings[0]]);
   });
 
+  it('allows proxy', async () => {
+    @injectable({tags: {service: true}})
+    class MyService {
+      hello() {
+        return 'Hello';
+      }
+    }
+    server.add(createBindingFromClass(MyService));
+    const view = new ContextView(server, filterByTag('service'), undefined, {
+      asProxyWithInterceptors: true,
+    });
+    const values = await view.values();
```

Comment:
The `expect` is inside the loop, what if the values are not coming out as an Iterable object?

Reply:
The values of a view will be always an array.